#!/usr/bin/env python
PACKAGE = "office_bot"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("vel_x", double_t, 0, "minimum velocity in x direction",  0, 0, 5)
gen.add("vel_y", double_t, 0, "minimum velocity in y direction",  0, 0, 5)
gen.add("vel_z", double_t, 0, "minimum velocity in z direction",  0, 0, 5)
gen.add("ang_vel_x", double_t, 0, "minimum angular velocity in x direction",  0, 0, 10)
gen.add("ang_vel_y", double_t, 0, "minimum angular velocity in y direction",  0, 0, 10)
gen.add("ang_vel_z", double_t, 0, "minimum angular velocity in z direction",  0, 0, 10)

exit(gen.generate(PACKAGE, "set_minimum_vel", "CmdVelLimits"))
